python/paddle/trainer_config_helpers/layers.py

```diff
@@ -5750,7 +5752,7 @@ def cross_entropy_with_selfnorm(input,
     :type input: LayerOutput.
     :param label: The input label.
     :type input: LayerOutput.
-    :param name: The name of this layers. It is not necessary.
+    :param name: The name of this layer. It is not necessary.
```

Comment:
please change the sentence "It is not necessary." into "It is not required."
The same below.